Parsers/ASimAuthentication/Parsers/ASimAuthenticationAADSigninLogs.yaml

```diff
@@ -15,23 +15,27 @@ References:
 Description: |
   This ASIM parser supports normalizing Azure Active Directory Interactive sign in logs, stored in the  SigninLogs table, to the ASIM Authentication schema.
 ParserName: ASimAuthenticationSigninLogs
+EquivalentBuiltInParser: _ASim_Authentication_SigninLogs
```

Comment:
@KustoKing Are these changes due to positions changing, is there anything added new or got removed?
Other than EquivalentBuiltInParser property?